src/main/java/com/alipay/oceanbase/rpc/location/LocationUtil.java

```diff
@@ -476,6 +478,35 @@ private static void getObVersionFromRemote(Connection connection)
         }
     }
 
+    // check tenant exist or not
+    private static void CheckTenantExistFromRemote(Connection connection, TableEntryKey key)
+                                                                     throws ObTableEntryRefreshException {
+        PreparedStatement ps = null;
+        ResultSet rs = null;
+        try {
+            ps = connection.prepareStatement(OB_TENANT_EXIST_SQL);
```

Comment:
```suggestion
            try (PreparedStatement ps = connection.prepareStatement(query)) {
```
Also remember to remove finally in the end of function

Reply:
edit done